File: python/fibonacci.py
def fibonacci(num):
    if (num == 1):
        return 0
    elif num == 2:
        return 1
    return fibonacci(num - 1) + fibonacci(num - 2)

print(fibonacci(1))
print(fibonacci(5))
print(fibonacci(8))

def GenerateFibonacciSequence(num):
    sequence = []
    for i in range(1, num + 1):
        sequence.append(fibonacci(i))
    return sequence

print(GenerateFibonacciSequence(8))

File: C#/Fibonacci.cs
namespace Fibonacci;

public class Fibonacci_Recursive
{
    public static List<int> GenerateFibonacciSequence(int num)
    {
        var sequence = new List<int> { };

        for (int i = 1; i <= num; i++)
        {
            sequence.Add(Fibonacci(i));
        }
        return sequence;
    }
    public static int Fibonacci(int n)
    {
        if (n == 1)
            return 0;
        if (n == 2)
            return 1;

        return Fibonacci(n - 1) + Fibonacci(n - 2);
    }
}